src/staticmaps.js

```diff
@@ -298,62 +301,43 @@ class StaticMaps {
     });
   }
 
-  drawLines() {
-    return new Promise(async (resolve) => {
-      if (!this.lines.length) resolve(true);
+  async drawLines() {
+    if (!this.lines.length) return true;
+    const chunks = chunk(this.lines, LINE_RENDER_CHUNK_SIZE);
```

Comment:
You could use bluebird's map with concurrency as well, it might be even faster http://bluebirdjs.com/docs/api/promise.map.html